src/UI/Routing/DataRouter.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { createBrowserRouter, RouterProvider } from "react-router";
+
+interface Props {
+  basename?: string;
+  children: React.ReactNode;
+}
+
+export function DataRouter({ basename, children }: Props) {
```

Comment:
it seems like this isn't used, but I could miss something

Reply:
yeah, nice catch, needed it in the v6 part, but not in the v7 anymore